playwright.config.ts

```diff
@@ -0,0 +1,27 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test';
+import * as dotenv from 'dotenv';
+
+dotenv.config(); // Loads variables from .env
+dotenv.config({ path: '.env.local' }); // Overrides or adds variables from .env.local
```

Comment:
can this not be hardcoded to '.env.local'

i remember something about nextjs not supporting .local and my local is actually called '.env.development'

can it be wildcard or support multiple .env ?

Reply:
oh yes, this should actually be loading `env.test`, so we can all keep it distinct from dev. i'll update it.